test/torchaudio_unittest/transforms/autograd_test_impl.py

```diff
@@ -76,6 +76,27 @@ def test_spectrogram(self, kwargs):
         waveform = get_whitenoise(sample_rate=8000, duration=0.05, n_channels=2)
         self.assert_grad(transform, [waveform], nondet_tol=1e-10)
 
+    @parameterized.expand([
+        ({'pad': 0, 'normalized': False, 'power': None, 'return_complex': True}, ),
+        ({'pad': 3, 'normalized': False, 'power': None, 'return_complex': True}, ),
+        ({'pad': 0, 'normalized': True, 'power': None, 'return_complex': True}, ),
+        ({'pad': 3, 'normalized': True, 'power': None, 'return_complex': True}, ),
+        ({'pad': 0, 'normalized': False, 'power': None}, ),
+        ({'pad': 3, 'normalized': False, 'power': None}, ),
+        ({'pad': 0, 'normalized': True, 'power': None}, ),
+        ({'pad': 3, 'normalized': True, 'power': None}, ),
+    ])
+    def test_inverse_spectrogram(self, kwargs):
+        # create a realistic input:
+        transform = T.Spectrogram(**kwargs)
```

Comment:
Similarly, it's ideal to not use `T.Spectrogram`, but we need to update the interface of [`get_spectrogram`](https://github.com/pytorch/audio/blob/9078c0b99e642943c193ee22b266f47fc21eda31/test/torchaudio_unittest/common_utils/data_utils.py#L115) for that.